api/swagger/swagger.yaml

```diff
@@ -529,6 +559,55 @@ definitions:
         type: string
         format: date-time
 
+  TrialPublication:
```

Comment:
As far as I can see, `TrialPublication` isn't being used anywhere, so you can delete it